src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -341,7 +341,8 @@ LedgerMaster::setValidLedger(std::shared_ptr<Ledger const> const& l)
     if (!standalone_)
     {
         auto validations = app_.validators().negativeUNLFilter(
-            app_.getValidations().getTrustedForLedger(l->info().hash));
+            app_.getValidations().getTrustedForLedger(
```

Comment:
Like comment above, a unit test for this is appropriate. Especially since this is the area with the bigger potential problem. What happens if a bogus sequence number ends up in a validation?

I'm not sure how many tests are needed, whether it's 1 per caller of getTrustedForLedger(), or if it's possible to have fewer tests to prove the point.